### PR TITLE
Turn on SDL for Arcade-Services

### PR DESCRIPTION
**Pull request by sunandabalu**
Test Build - https://dev.azure.com/dnceng/internal/_build/results?buildId=243667&view=results

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -196,3 +196,16 @@ stages:
       # Symbol validation isn't being very reliable lately. This should be enabled back
       # once this issue is resolved: https://github.com/dotnet/arcade/issues/2871
       enableSymbolValidation: false
+      # This is to enable SDL runs part of Post-Build Validation Stage
+      SDLValidationParameters:
+        enable: true
+        params: ' -SourceToolsList @("policheck","credscan")
```
Comment (jcagme) [inline azure-pipelines.yml]: >  [](start = 17, length = 1)

Nit: remove space?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -196,3 +196,16 @@ stages:
...
+        -TsaInstanceURL "https://devdiv.visualstudio.com/"
+        -TsaProjectName "DEVDIV"
+        -TsaNotificationEmail "subal@microsoft.com"
+        -TsaCodebaseAdmin "REDMOND\\subal"
+        -TsaBugAreaPath "DevDiv\\NET\\NET Core "
```
Comment (jcagme) [inline azure-pipelines.yml]: >  [](start = 46, length = 1)

Nit: remove space?

**Comment by jcagme** [comment]
Is there a test build with this on to validate things work as expected?

**Comment by sunandabalu** [comment]
> Is there a test build with this on to validate things work as expected?

Link is part of the description :)

**Comment by jcagme** [comment]
Of course :)

---
In reply to: [506888025](https://github.com/dotnet/arcade-services/pull/499#issuecomment-506888025) [](ancestors = 506888025)

**Pull request merged**